doc/man3/SSL_CIPHER_get_name.pod

```diff
@@ -109,7 +109,7 @@ cipher B<c>.
 
 SSL_CIPHER_description() returns a textual description of the cipher used
 into the buffer B<buf> of length B<len> provided.  If B<buf> is provided, it
-must be at least 128 bytes, otherwise a buffer will be allocated using
+must be at least 128 bytes. If no buf is provided one will will be allocated using
```

Comment:
It would be good to be consistent with the rest of the file, it already uses multiple different ways of saying the same thing.. i.e
If B<X> is NULL
If there is no B<X>
 
e.g
If B<buf> is not NULL, the length B<len> must be at least 128 bytes. If B<buf> is NULL a buffer will be allocated internally.

Reply:
@slontis I am afraid any more substantial changes would make this non-trivial PR.

Reply:
However there is double `will` and I also prefer `If B<buf> is NULL it will ...`

Reply:
> I also prefer If B<buf> is NULL it will ...

I've adjusted to adopt this preferred description.